src/main/java/com/redhat/pantheon/sling/PantheonRepositoryInitializer.java

```diff
@@ -26,10 +28,16 @@
  * Created by ben on 3/7/19.
  */
 @Component(service = SlingRepositoryInitializer.class)
-public class PantheonInitializer implements SlingRepositoryInitializer {
+public class PantheonRepositoryInitializer implements SlingRepositoryInitializer {
+
+    private static final Logger log = LoggerFactory.getLogger(SlingRepositoryInitializer.class);
 
     @Override
     public void processRepository(SlingRepository slingRepository) throws Exception {
+        initializeRepositoryACLs(slingRepository);
+    }
+
+    private void initializeRepositoryACLs(SlingRepository slingRepository) throws RepositoryException {
```

Comment:
What was the purpose of pulling this out into its own method? Just for readability's sake?

Reply:
to separate the different tasks that we would be doing when initializing the repository. Right now it's just creating some ACLs, but there may be more in the future. I did this refactoring originally when I had the code to load gem paths in this class.